spec/services/profiles/edit_service_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ::Profiles::EditService do
+  subject(:service) do
+    described_class.call(profile, params)
+    profile.reload
+  end
+
+  let(:profile)         { create(:profile) }
+  let(:updated_surname) { 'Updated surname' }
+  let(:updated_phone)   { '0987654321' }
+
+  let(:params) do
+    {
+      surname: updated_surname,
```

Comment:
alignign:
```
{
  surname: updated_surname,
  phone:   updated_phone
}
```

Reply:
Rubocop is falling if I am doing like that.  Error = 'Align the keys of a hash literal if they span more than one line.
      phone:   updated_phone'